Sources/HaCWebsiteLib/Events/ExternalWebsiteEvent.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+struct ExternalWebsiteEvent : Event {
```

Comment:
As this is supposed to be referring to an external website, should it have a URL pointing to said external website?